soapfast/scripts/list_spherical_components.py

```diff
@@ -28,7 +28,7 @@
 if rank == 0:
     tens = [str(ftrs[i].info[args.property]) for i in range(len(ftrs))]
 elif rank == 2:
-    tens = [' '.join(np.concatenate(ftrs[i].info[args.property]).astype(str)) for i in range(len(ftrs))]
+    tens = [' '.join(np.concatenate(ftrs[i].info[args.property][np.newaxis,:]).astype(str)) for i in range(len(ftrs))]
```

Comment:
This is fine, if it works; it's not a script I use often (or at all) so hadn't checked